backend/src/admin-app.ts

```diff
@@ -30,6 +30,7 @@ import codeRouter from './v1/routes/code-routes';
 import announcementsRoutes from './v1/routes/announcement-routes';
 import analyticRoutes from './v1/routes/analytic-routes';
 import resourcesRoutes from './v1/routes/resources-routes';
+import dashboardMetricsRoutes from './v1/routes/dashboard-metrics-routes';
```

Comment:
make the folder structure for routes to be routes/dashboard - followed by individual file names - announcement-metrics